src/formatDate.js

```diff
@@ -8,7 +8,40 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  // Створюємо копії fromFormat і toFormat, щоб не змінювати параметри функції
+  const fromFormatCopy = [...fromFormat];
+  const toFormatCopy = [...toFormat];
```

Comment:
Creating a copy of the fromFormat and toFormat arrays is unnecessary since these arrays are not being modified. You can directly use the original arrays without copying them, which will make the code simpler and save memory.